### PR TITLE
NO-ISSUE - Update docs to say openshift 4.6 instead of 4.5

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

File: GUIDE.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ As `$USER` user with `sudo` privileges,
 
         [$USER@assisted_installer assisted-test-infra]# make create_full_environment
 
-1.  **(Optional)** Currently the Installer defaults to deploying OpenShift 4.5. If you wish to change this value, set `OPENSHIFT_VERSION` to a different value, e.g. 4.6
+1.  **(Optional)** Currently the Installer defaults to deploying OpenShift 4.6. If you wish to change this value, set `OPENSHIFT_VERSION` to a different value, e.g. 4.7
```
Comment (omertuc) [inline GUIDE.md:45]: Not entirely sure about this one. Is 4.7 even currently supported? Should consider removing this sentence altogether if we don't.

Comment (YuviGold) [reply GUIDE.md:45]: Yes it does.

**Review by YuviGold — APPROVED** [review]
LGTM

**Pull request merged**